### DISCUSSION TITLE
Several issues with Python 3.12.4 version install

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [actions/virtual-environments#10029] [General questions] [answered]
Hi Team, 

Im getting several issues with python 3.12.4 version install in both Linux and windows builds. please find the error info below.

Windows:
![image](https://github.com/actions/runner-images/assets/113025693/0569ace1-7c16-4fb2-bd1b-a1a1bb38b124)

Linux: Path issue
![image](https://github.com/actions/runner-images/assets/113025693/178ea816-4407-450e-b876-8956a7cd0ac4)

![image](https://github.com/actions/runner-images/assets/113025693/e8b7c5bd-4563-4958-b089-ac6a8f812476)

May you please have a look and solve the issue.

i look forward for your response.

**Comment by LokeshNagireddy** [comment]
Hi @erik-bershel , May you please have a look at this issue and let me know if there is any fix already done for this issue.

**Comment by erik-bershel** [comment, marked as answer]
External issue:
https://github.com/actions/setup-python/issues/887

**Comment by LokeshNagireddy** [comment]
issue solved
